cmd/flux/main.go

```diff
@@ -138,30 +142,17 @@ func NewRootFlags() rootFlags {
 
 func main() {
 	log.SetFlags(0)
-	configureKubeconfig()
-	configureDefaultNamespace()
 	if err := rootCmd.Execute(); err != nil {
 		logger.Failuref("%v", err)
 		os.Exit(1)
 	}
 }
 
-func configureKubeconfig() {
-	switch {
-	case len(rootArgs.kubeconfig) > 0:
-	case len(os.Getenv("KUBECONFIG")) > 0:
-		rootArgs.kubeconfig = os.Getenv("KUBECONFIG")
```

Comment:
Does cli-runtime reads `KUBECONFIG` from env?

Reply:
it uses `clientcmd.NewDefaultClientConfigLoadingRules()` to load the config, which includes the `KUBECONFIG` env var.

Reply:
But I don't think it gets added to `kubeconfigArgs.KubeConfig`, it's just respected when calling `kubeconfigArgs.ToRESTConfig()`.